docs/opencompetition.md

```diff
@@ -0,0 +1,116 @@
+
+## Dates: 
+		Submission deadline is June 1, 2020, AoE
+
+## Awards:
+The type of awards, if any, will be announced later.
+
+## About Nevergrad and IOHprofiler
+### Nevergrad
+Nevergrad is an open source platform for derivative-free optimization:  it contains a wide range of optimizers, test cases, supports multiobjective optimization and handles constraints. It automatically updates results of all experiments merged in the codebase, hence users do not need computational power for participating and getting results.
+
+### IOHProfiler
+IOHprofiler is a tool for benchmarking iterative optimization heuristics such as local search variants, evolutionary algorithms, model-based algorithms, and other sequential optimization techniques. IOHprofiler has two components: 
+- IOHexperimenter for running empirical evaluations
+- IOHanalyzer for the statistical analysis and visualization of the experiment data.
+A documentation is available here: https://iohprofiler.github.io/  
+
+## Motivation
+We want to build open-source, user-friendly, and community-driven platforms for comparing different optimization techniques. Our key principles are reproducibility, open source, and ease of access. While we have set some first steps towards such platforms, we believe that the tools can greatly benefit from the contributions of the various communities for whom they are built.
+
+## Award Committee Members
+The award committee members are:
+- Enrique Alba (University of Málaga, Spain)
+- Maxim Buzdalov (ITMO University, Russia)
+- Josu Cerebio (University of the Basque country, Spain)
+- Benjamin Doerr (Ecole Polytechnique, France)
+- Tobias Glasmachers (Ruhr-Universität Bochum, Germany)
+- Manuel Lopez-Ibanez (University of Manchester, UK)
+- Katherine Mary Malan  (University of South Africa)
+- Luís Paquete (Coimbra, Portugal)
+- Jan van Rijn (Leiden University, The Netherlands)
+- Marc Schoenauer (Inria, France)
+- Thomas Weise (Hefei University, China)
+
+
+Policy re. possible conflict of interest: Award committee members can not propose as winner a person with whom they worked directly during the previous 12 months. There is no restriction for coworkers of other committee members than oneself.
+Submissions made by members of the organizing committee or by employees of Facebook can not be awarded. Their coworkers can be rewarded (but only for work that does not involve organizers nor Facebook employees).
+
+## Details about the Submissions 
+All submissions are based on pull request, which are directly made to either one of the tools, via
+https://github.com/facebookresearch/nevergrad for Nevergrad 
+https://github.com/IOHprofiler/ for IOHprofiler. For submissions to IOHprofiler, please use the competition branch 
+
+## Recommended topics (non-exhaustive list)
+We identify the following list of topics for which we feel that great contributions are possible.
+##Improvements in any of the following optimization categories:
```

Comment:
skip a line before the title, and a space after the `#` , otherwise it does not render well, also, a `#` is missing

Reply:
Done.